From: Mathieu Malaterre <malat@debian.org>
Subject: Typo in PHP find module
Forwarded: yes
Origin: other
Bug-Debian: http://bugs.debian.org/cgi-bin/bugreport.cgi?bug=722475
Reviewed-by: Modestas Vainius <modax@debian.org>

There is a missing CACHE keyword which make the set() command produce improper
result in FindPHP4

--- a/Modules/FindPHP4.cmake
+++ b/Modules/FindPHP4.cmake
@@ -40,7 +40,7 @@ if(PHP4_FOUND_INCLUDE_PATH)
   foreach(php4_path Zend main TSRM)
     set(php4_paths ${php4_paths} "${PHP4_FOUND_INCLUDE_PATH}/${php4_path}")
   endforeach()
-  set(PHP4_INCLUDE_PATH "${php4_paths}" INTERNAL "PHP4 include paths")
+  set(PHP4_INCLUDE_PATH "${php4_paths}" CACHE INTERNAL "PHP4 include paths")
 endif()
 
 find_program(PHP4_EXECUTABLE NAMES php4 php )
